### PR TITLE
Codecov is back

### PR DESCRIPTION
**Pull request by RingerJK**
### Description
Codecov is back

### CODE REVIEW

**Review by VysotskiVadim — APPROVED** [review]
Amazing ❤️

File: .circleci/config.yml [hunk 1]
```diff
@@ -282,11 +282,14 @@ commands:
...
-  codecov:
+  codecov-upload:
     steps:
       - run:
           name: Post code coverage reports to Codecov.io
-          command: pip3 install --user codecov && /root/.local/bin/codecov
+          command: |
+            curl -Os https://uploader.codecov.io/latest/linux/codecov
+            chmod +x codecov
+            ./codecov
```
Comment (VysotskiVadim) [inline .circleci/config.yml:292]: A thought I got after approving. Is it safe to always download script and run it? I guess pip3 somehow verifies that the script is provided by original author(sorry, I didn't explore pip3 but other package managers do so). With a simple download we can be attacked if codecov temporally loose control over their domain.

Comment (RingerJK) [reply .circleci/config.yml:292]: The current repo is open-source, I suppose we might lose some tokens only, but in this case we have to reset them. For private repo uses codecov token.

Comment (RingerJK) [reply .circleci/config.yml:292]: thanks, got the idea. 
> curl -Os https://uploader.codecov.io/latest/linux/codecov

this is recommended way how to get codecov uploader https://docs.codecov.com/docs/codecov-uploader#using-the-uploader-with-codecovio-cloud. 

We might use checking the control sum of codecov binary https://docs.codecov.com/docs/codecov-uploader#using-the-uploader-with-codecovio-cloud. I'll cut separate ticket

**Pull request merged**